operations/network/management_network/bican_technical_details.md

```diff
@@ -0,0 +1,252 @@
+# Bifurcating the CAN - CSM 1.2 Feature Details
+
+1. [CAN new features overview](#1-can-new-features-overview)
+2. [High Speed CAN (CHN)](#2-high-speed-can-chn)
+    1. [CHN system ingress endpoints accessible in CSM 1.2](#21-chn-system-ingress-endpoints-accessible-in-CSM-12)
+    2. [CHN system egress endpoints accessible in CSM 1.2](#22-chn-system-egress-endpoints-accessible-in-CSM-12)
+    3. [Endpoint Naming](#23-endpoint-naming)
+        1. [Touchpoints: effects and changes](#231-touchpoints-effects-and-changes)
+        2. [When naming occurs](#232-when-naming-occurs)
+        3. [Ability to change post-install](#233-ability-to-change-post-install)
+    4. [Endpoint addressing](#24-endpoint-addressing)
+        1. [Touchpoints: effects and changes](#241-touchpoints-effects-and-changes)
+        2. [When addressing occurs](#242-when-addressing-occurs)
+        3. [Ability to change post-install](#243-ability-to-change-post-install)
+    5. [Traffic separation and routing](#25-traffic-separation-and-routing)
+        1. [Touchpoints: effects and changes](#251-touchpoints-effects-and-changes)
+        2. [When configuration occurs](#252-when-configuration-occurs)
+        3. [Ability to change post-install](#253-ability-to-change-post-install)
+3. [Management CAN (CMN)](#3-management-cancmn)
+    1. [Traffic separation and routing](#31-traffic-separation-and-routing)
+    2. [Endpoint naming](#32-endpoint-naming)
+    3. [Endpoint addressing](#33-endpoint-addressing)
+    4. [Changes](#34-changes)
+        1. [Touchpoints: effects and changes](#341-touchpoints-effects-and-changes)
+        2. [When configuration occurs](#342-when-configuration-occurs)
+        3. [Ability to change post install](#343-ability-to-change-post-install)
+4. [CAN external/site access (site DNS, LDAP, etc.)](#4-can-externalsite-access-site-dns-ldap-etc)
+    1. [Traffic separation and routing](#41-traffic-separation-and-routing)
+    2. [Changes](#42-changes)
+        1. [Touchpoints: effects and changes](#421-touchpoints-effects-and-changes)
+        2. [When configuration occurs](#422-when-configuration-occurs)
+        3. [Ability to change post-install](#423-ability-to-change-post-install)
+
+## 1 CAN new features overview
+
+Bifurcation or splitting of the Customer Access Network (CAN) enables customization of customer traffic to and from the system.
+Customization will be performed during installation.
+For CSM 1.2 there are two new customer access networks being introduced as part of the process to split the existing monolithic CAN:
+
+1. **High Speed CAN - CHN (Customer High Speed Network)** : This feature adds the ability to connect to User Application Nodes (UAN), UAI, Compute Nodes
+and Kubernetes API endpoints from the customer site via the High Speed Network (HSN).
+2. **Management CAN - CMN (Customer Management Network)** :  Using a new VLAN on the Management Network, this feature allows system administrative access from the customer site.
```

Comment:
IMO, the abbreviation `CAN` is being used in this PR as both a type of network as well as an instance of a network -- and this is confusing (at least to me :)). This causes the following issues:

1. CHN is called out but not the new CAN, which is an alternative to CHN. From a reader's perspective, the "High Speed CAN - CHN" either only calls out CHN only, or it bundles together new CAN and CHN but erroneously calls new CAN "high speed". We don't have to refer to CHN as "High Speed CAN - CHN", instead can simply refer to its full form "Customer High-speed Network (CHN)". This would allow us to also mention the "new 1.2 CAN". Please see an example of this verbiage at the end of this comment.

2. Same goes for not using "Management CAN - CMN" and instead just using "Customer Management Network (CMN)". It would reduce the amount of technical terms and help us differentiate between pre-1.2 CAN and new CMN.

3. Also, I don't think CMN is "using a new VLAN on the Management Network" ... I thought CMN VLAN is the same pre-1.2 CAN VLAN, and CHN/new-CAN are new VLANs?

------

I am attaching a summary of BICAN here that I used for the upgrade process that highlights what was there pre-1.2 and what exact networks will be introduced as part of the 1.2 upgrade:


```
Prior to CSM 1.2, the single Customer Access Network (CAN) carried both the admin network traffic and the user network traffic. CSM 1.2 introduces bifurcated CAN (BICAN), which is designed to separate admin network traffic and user network traffic.

With BICAN, the pre-1.2 CAN network is split into two separate networks:

1. Customer Management Network (CMN)

   This network allows only system administrative access from the customer site. The pre-1.2 CAN is renamed to CMN. <rest of docs pertaining to upgrade snipped>

1. Customer Access Network (CAN) / Customer High-speed Network (CHN)

   For user traffic only (e.g. users running and monitoring jobs), CSM 1.2 allows you to pick one of two networks:

      - Customer Access Network (CAN): this is a new network (VLAN6 in switches) that runs over the management network. This
         network must not be confused with pre-1.2 CAN, which was a monolithic network that allowed both user and administrative
         traffic, was configured as VLAN7 in switches, and is now renamed to CMN. The new CAN allows only user traffic.

      - Customer High-speed Network (CHN): this is a new network (VLAN5 in switches) that runs over the high-speed fabric.

   You must only pick either the new CAN or CHN, but not both. The rest of the installation guide will provide you with options for configuring either the new CAN or CHN.

Pivoting the pre-1.2 CAN to the new CMN allows administrative traffic (already on the pre-1.2 CAN) to remain as-is while moving standard user traffic to a new site-routable network.
```

Reply:
Since this is my IMO, please feel free to resolve this comment without any action :)

Reply:
1. Looks good 
2. Can you please make the similar change in bican_technical_details.md? It still says "High Speed CAN - CHN" etc.
3. See point (3) in original comment about CMN not being a "new VLAN". If it is, in fact a new VLAN, please ignore this comment.

Reply:
i think i have item #2 sorted.  please have a look and let me know if i missed any spots.